src/game_inventory.cpp

```diff
@@ -783,6 +787,10 @@ class activatable_inventory_preset : public pickup_inventory_preset
                 }
             }
 
+            if( ( *loc ).is_medication() && !p.can_use_heal_item( ( *loc ).typeId() ) ) {
```

Comment:
```suggestion
            if( it.is_medication() && !p.can_use_heal_item( it.typeId() ) ) {
```
So check above at the top of the function: `loc` may refer to a container *with* medication. Therefor `it` (the actual medial item) is extracted from it. Your code would just check the container item.